### PR TITLE
Fix/update tutorials notebooks

### PR DESCRIPTION
**Pull request by clement-pages**
This PR updates `pyannote.audio` tutorials to version `3.1.1`. This update also contains the elements enabling a user to run these tutorials on `Google Colab`.

### CODE REVIEW

**Comment by hbredin** [comment]
Can you take care of the conflict in `add_your_own_task` notebook?

File: tutorials/add_your_own_model.ipynb
Comment (hbredin) [inline tutorials/add_your_own_model.ipynb:1]: Missing [newly added](https://github.com/pyannote/pyannote-audio/commit/66dd72bb2b807aaf6d011c89678d85b51fb3b859) `num_frames` and `receptive_field` methods that must be defined.

Comment (hbredin) [inline tutorials/add_your_own_model.ipynb:1]: Also, I don't think we need to send the model to GPU nor set the accelerator manually in the latest version of lightning. (I think) you can safely remove this part.

Comment (clement-pages) [reply tutorials/add_your_own_model.ipynb:1]: Done in [1e5cd2f](https://github.com/pyannote/pyannote-audio/pull/1609/commits/1e5cd2fcab852012bd89fabb19955e9c296a5860)

**Comment by hbredin** [comment]
This PR now has conflicts.

File: tutorials/voice_activity_detection.ipynb
Comment (hbredin) [inline tutorials/voice_activity_detection.ipynb:1]: The text says "for the purpose of this tutorial we train for just 2 epochs" but `max_epochs` is set to 1.

Can you make another pass, double check and fix this kind of issues on all tutorials?

**Comment by hbredin** [comment]
Merged but I think there is still some work needed on this tutorials.
For instance (and I did not go over everything, far from it), this sentence "The pipeline will automatically use GPUs when available. On CPU it might take a long while (up to 10x RT)." is no longer correct in 3.x

Pipelines needs to be send to device with `pipeline.to(torch.device(...))`

**Pull request merged**